api/db/changelog/db.changelog-244-add-lab-column-to-workspace.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet author="tarekahmed" id="db.changelog-244-add-lab-column-to-workspace">
+        <addColumn tableName="workspace">
+            <column name="lab" type="ENUM('AOU', 'VWB')">
```

Comment:
What does the column `lab` mean?  feels it can be confusing.
If we just want to tell whether a workspace owned by Terra or VWB,  a boolean is_vwb_workspace is better than using enum. It's easier to use in java code,  easier to deprecreate, and lighter-weighted in sql

Reply:
I wanted this to be extensible to account for additional "labs" in the future. This is based on DGlazer's ideas where he mentioned that we may want this in the future. I am not sure if it's applicable in the near future but I wanted to keep our options open.

Reply:
We discussed this on slack before, for the sake of simplicity I'm happy to switch it back to a boolean. That was my original design earlier but as I said I wanted it more extensible.

Reply:
enum also works if we think there might be future extension. My main concern is the confusing, using enum is okay, but we might need to come up a better name / description somewhere. 
Like asked above, what is `lab`, what does `lab=aou` mean?

Reply:
Just to close this thread, as we discussed in slack. Let's keep it simple now by having this as a boolean. If the need arises to add more labs we can revisit.
Also we agreed we don't have a good naming convention for the legacy AoU vs the VWB one so maybe if needed in the future we can have a platform column